### ISSUE TITLE
File import not working for docker-compose

### ISSUE THREAD

**Issue by Atharex** [heartexlabs/label-studio#710]
**Describe the bug**
I'm running the default v1.0 docker-compose setup and trying to import a jpg file for image classification. 
Any image I try, receive the error: 
`Cannot load image (/data/upload/my-image.jpg). Check console/network panel for more info.`

This error does not occur on the single image docker. 

**Expected behavior**
Image is correctly loaded and available also on docker compose

**Environment (please complete the following information):**
 - Label Studio Version v1.0

**Comment by niklub** [comment]
Hi, @Atharex ! Please try out the RC branch https://github.com/heartexlabs/label-studio/pull/681 Hope it has been  fixed there

**Comment by Atharex** [comment]
Hi @niklub the docker-compose is currently also not functioning in the RC branch. 
There it seems the script is missing in the docker image

```
Recreating c48fb39c0f51_label-studio_app_1 ... error

ERROR: for c48fb39c0f51_label-studio_app_1  Cannot start service app: OCI runtime create failed: container_linux.go:344: starting container process caused "exec: \"./deploy/wait-for-postgres.sh\": stat ./deploy/wait-for-postgres.sh: no such file or directory": unknown

ERROR: for app  Cannot start service app: OCI runtime create failed: container_linux.go:344: starting container process caused "exec: \"./deploy/wait-for-postgres.sh\": stat ./deploy/wait-for-postgres.sh: no such file or directory": unknown
ERROR: Encountered errors while bringing up the project.
```

**Comment by makseq** [comment]
@Atharex Could you check it again from the latest commit of the master branch?

**Comment by Atharex** [comment]
Seems it works now, I can see the images.

Though in some cases when importing multiple images I get the following error:
`JSON.parse: unexpected character at line 1 column 1 of the JSON data`

But at least the imported images that work are visible now +1

**Comment by makseq** [comment]
@Atharex What way do you import multiple images?

**Comment by Atharex** [comment]
Through the UI. I click on the UI for the file browser and then select multiple files
![data-import-ls](https://user-images.githubusercontent.com/2872241/114173737-eccc6000-9937-11eb-921b-5435b82223d6.png)

**Comment by niklub** [comment]
@Atharex please feel free to create another issue with multiple images importing issue, hope this one related to the docker-compose installation has been fixed after `label-studio>=1.0.1`
